lib/fsm.c

```diff
@@ -850,10 +852,21 @@ int rpmPackageFilesInstall(rpmts ts, rpmte te, rpmfiles files,
     char *tid = NULL;
     const char *suffix;
     char *fpath = NULL;
+    Header h = rpmteHeader(te);
+    const char *payloadfmt = headerGetString(h, RPMTAG_PAYLOADFORMAT);
+    bool cpio = true;
```

Comment:
Rpm uses 0/1 integers for booleans throughout. While C99 is fine as such, to me this lone "bool" only ends up looking out of place.

Reply:
Addressed in https://github.com/rpm-software-management/librepo/pull/222